content/decisions-with-conditionals/studio/_index.md

```diff
@@ -0,0 +1,75 @@
+---
+title: "Studio: Shuttle Launch"
+date: 2023-05-25T12:55:09-05:00
+draft: false
+weight: 3
+originalAuthor: John Woolbright # to be set by page creator
+originalAuthorGitHub: jwoolbright23 # to be set by page creator
+reviewer: Sally Steuterman # to be set by the page reviewer
+reviewerGitHub: gildedgardenia # to be set by the page reviewer
+lastEditor: # update any time edits are made after review
+lastEditorGitHub: # update any time edits are made after review
+lastMod: # Wed Jul 5 08:49:19 AM CDT 2023
+---
+
+In this studio, you are going to write code to verify that your Space Shuttle is prepared for LiftOff! This activity will combine knowledge you have gained from this chapter and the [Data and Variables Chapter]({{< relref "../../data-and-variables/_index.md" >}})
+
+{{% notice blue Note "rocket" %}}
+The starter code for this studio can be found inside of the following repository: [javascript-projects](https://github.com/LaunchCodeEducation/javascript-projects/) within the `booleans-and-conditionals` directory.
```

Comment:
The javascript-projects repository doesn't yet have a booleans-and-conditional directory.

Reply:
content exists it just has not been merged into the main branch quite yet.